didx-xyz/yoma-mobile
# YOMA-592 - feat(MyCv): widget navigation

## Scope [YOMA-592](https://yomaworld.atlassian.net/browse/YOMA-592)

![](https://media.giphy.com/media/Bd22EXOTCRRhS/giphy.gif)
## Work Done
- switching most of the widget icons to (+) instead of edit.
- adding onPress for the widget items - setting both widget items and view all to the 'view' section
- done for all widgets
- removed the 'completed courses' module as they're unused. 
- NOTE: haven't updated the **About** widget, because we need to make the View page first - so will do that at the same time.

## Steps to Test
- open each of the widgets -  tap on the view all, a line item and expect to land on the Views, press the + and and on the add items.
